integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/server/conf/axis2/axis2.xml

```diff
@@ -255,29 +255,29 @@
     check com.sun.mail.pop3 and com.sun.mail.imap package documentation for more details-->
     <!--</transportReceiver>-->
 
-    <!--Uncomment this and configure as appropriate for JMS transport support, after setting up your JMS environment (e.g. ActiveMQ)
+    <!--Uncomment this and configure as appropriate for JMS transport support, after setting up your JMS environment (e.g. ActiveMQ) -->
```

Comment:
Do we need this line anymore since we have now uncommented the test cases?

Reply:
Yes that is needed to run the ESBJAVA4279_MPRetryUponResponseSC_500_withNonRetryStatusCodes_200_and_202_TestCase testcase

Reply:
Sorry I didn't see that this is axis2.xml